### DISCUSSION TITLE
SKR Mini E3 V3 Questions

### DISCUSSION THREAD

**Discussion by Moobien** [mriscoc/Ender3V2S1#637] [Q&A]
Hey Folks, I got my SKR Mini E3 V3 installed today and I have a few questions I was wondering if the community could help me with.  I will say in advance that I have NEVER compiled firmware and have absolutely no idea how to so please if you help me, treat me like a luddite lol.

Questions:

1) This board has a neopixel port.  Is there any difference between NeoPixel and RGB or are they interchangable? ie can use any RGB strip.  The main question for this is can I control this via firmware?

2) The board supports sensorless homing, does the firmware support this  also? (I probably won't use it myself)

3) The board has a third fan connector for the board cooling fan so all 3 fans have independant ports.  Is there any way to have control over this fan? 

3a) I did notice that when I turned on the printer with the new board the cooling fan for the hot end didn't seem to turn on.

4) I know I am probably gonna have to compile my own firmware for all this so is there any GOOD 'firmware for dummies' resources I can use to guide me through this?

**Comment by mriscoc** [comment]
Maybe you should ask here: https://github.com/mriscoc/Ender3V2S1/discussions/131
